### PR TITLE
Persist selected resources in the url search params so that urls are …

### PR DESCRIPTION
**Pull request by lukehollis**
…sharable

### Before submitting this PR, please use netlify to make sure:
- [x] The app looks okay on web
- [ ] The app looks okay on phone
- [x] Filter by category still works

### If you might be concerned about these things, also check:
- [ ] The app looks okay in ipad
- [ ] Search still works

### CODE REVIEW

**Comment by lukehollis** [comment]
Addresses https://github.com/codeforboston/communityconnect/issues/155

**Comment by lukehollis** [comment]
This is a minimal integration, but in the future, I think it'd be better to use a single ground of truth for selected resources state (in URL only instead of redux) to prevent future complications.

File: src/components/SavedResources/SavedResource.js [hunk 1]
```diff
@@ -30,6 +30,10 @@ class SavedResource extends Component {
     this.removalConfirmed = this.removalConfirmed.bind(this);
   }
 
+  componentDidMount() {
```
Comment (bran-van-d) [inline src/components/SavedResources/SavedResource.js]: Is this added with the intention to handle do something with it in the future ?

File: src/reducers/resourceReducer.js [hunk 2]
```diff
@@ -43,6 +46,23 @@ export function filteredResource(state = initialState.filteredResource, action)
...
+          const selectedResources = [];
+          selectedResourceIds.forEach(selectedResourceId => {
+            action.resource.forEach(resource => {
+              if (resource.id === selectedResourceId) {
+                selectedResources.push(resource);
+              }
+            });
+          })
+
+          return Object.assign([], state, selectedResources);
```
Comment (bran-van-d) [inline src/reducers/resourceReducer.js]: Does it cause any issue if this ends if `selectedResources` turns out to be undefined ?

**Pull request merged**